OCI-CLI_reference.md

```diff
@@ -0,0 +1,1218 @@
+
+  There are two types of aliases: *global aliases* and *command sequence aliases*.
+
+- [Specifying option aliases](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_Option_Aliases)
+
+  **Section:** OCI_CLI_PARAM_ALIASES  
+  **Format:** \<*option alias*\> = \<*original option*\>  
+
+  **Example:**  
+  \--ad = \--availability-domain
+
+  > **Note:** Option aliases are applied globally.  
+  {:.notice}
+
+- [Specifying named queries](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Specifying_Named_Queries)
+
+  If you use the **\--query** parameter to filter or manipulate the output, you can define named queries instead of using a **JMESPath** expression on the command line.  
+
+  **Section:** OCI_CLI_CANNED_QUERIES
+
+- autocomplete
+
+  > **Note:** If you used the **CLI installer**, autocomplete was already installed by default.  
+  >
+  > However, if you manually installed the CLI or only wish to enable autocomplete on a per-session basis, use one of the commands below.  
+  {:.notice}
+
+- **Manual installation -** Run the following command:  
+
+  ```sh
+  oci setup autocomplete
+  ```
+
+- **Session-by-session basis** -- Run the following command:
+
+  ```sh
+  eval "$(_OCI_COMPLETE=source oci)"
+  ```
+
+## Using the CLI
+
+[Reference documentation](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm)
+
+### Basic command line syntax
+
+Most commands must specify a service, followed by a resource type and then an action.  
+
+The basic command line syntax is:  
+
+**oci \<*service*\> \<*type*\> \<*action*\> \<*options*\>**
+
+**Example:**
+
+The following command shows a typical command line construct:  
+
+```sh
+oci compute instance launch --availability-domain "EMIr:PHX-AD-1" -c <compartmentID> --shape "VM.Standard1.1" --display-name "Instance 1 for sandbox" --image-id <imageID> --subnet-id <subnetID>
+```
+
+Here, **compute** is the *service*, **instance** is the *type*, **launch** is the *action*, and the balance of the command represents the *options*.  
+
+> **Note:** Throughout this document, we'll be using the term *oci* to refer to commands executed within the CLI environment. This is to be distinguished from *OCI*, which refers to the Oracle Cloud Infrastructure platform.  
+{:.notice}
+
+#### Getting help with commands
+
+For help with a specific command, you can enter **help *\<command\>*** on the command line or view the [Command Line Reference](https://docs.oracle.com/iaas/tools/oci-cli/latest/oci_cli_docs/).  
+
+In addition, you can get help for any command by using the **\--help**,
+**-h**, or **-?** flag.  
+
+**Examples:**
+
+- oci \--help
+
+- oci os bucket -h
+
+- oci os bucket create -?  
+  For a full list of commands, refer to the [OCI CLI help documentation](https://docs.oracle.com/iaas/tools/oci-cli/latest/oci_cli_docs/).  
+
+#### Output formats
+
+- **JSON (\--output JSON) -** (default) The output is formatted as JSON string.  
```

Comment:
Like this would make more sense to me as:

- **JSON** (`--output JSON`)

I'm not sure what the trailing hyphen after the closing paren is, either. Is that supposed to part of the command option in parenthesis, or just a formatting decision for the list item/description separation?

Reply:
Just a formatting decision. :) What I did do was pull the hyphen out of the bold formatting just to build a stronger visual connection with the text as opposed to the command option.